src/components/modals/add-network.tsx

```diff
@@ -2,18 +2,20 @@ import React from 'react';
 import { Box, color, ControlledModal, Flex, IconButton, Stack } from '@stacks/ui';
 import { IconX } from '@tabler/icons';
 import { Link, Text, Title } from '@components/typography';
-import { useModal } from '@common/hooks/use-modal';
+import { closeModal, getOpenedModal } from '@common/state/slices/modal';
```

Comment:
@He1DAr I thought we would prefer organising folders by features from now on.

Reply:
+1 on this

Wouldn't `common/state/modal/modal.slice.ts` be more organising by feature?

Reply:
I was under the impression modal is a global state, but looking now we can def colocate it with modal 👍